sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Demo.Tracing/Program.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using OpenTelemetry;
+using OpenTelemetry.Resources;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Demo.Tracing
+{
+    public class Program
+    {
+        private const string ConnectionString = "InstrumentationKey=00000000-0000-0000-0000-000000000000";
+
+        public static void Main()
+        {
+            GenerateTraces();
+            GenerateLogs();
+            GenerateMetrics();
+        }
+
+        private static void GenerateTraces()
+        {
+            ActivitySource source = new ActivitySource("MyCompany.MyProduct.MyLibrary");
+
+            var resourceAttributes = new Dictionary<string, object> { { "service.name", "my-service" }, { "service.namespace", "my-namespace" }, { "service.instance.id", "my-instance" } };
+            var resourceBuilder = ResourceBuilder.CreateDefault().AddAttributes(resourceAttributes);
+            using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+                            .SetResourceBuilder(resourceBuilder)
+                            .AddSource("MyCompany.MyProduct.MyLibrary")
+                            .AddAzureMonitorTraceExporter(o =>
+                            {
+                                o.ConnectionString = ConnectionString;
+                            })
+                            .Build();
+
+            using (var activity = source.StartActivity("SayHello"))
+            {
+                activity?.SetTag("foo", 1);
+                activity?.SetTag("baz", new int[] { 1, 2, 3 });
+                activity?.SetStatus(ActivityStatusCode.Ok);
+
+                using (var nestedActivity = source.StartActivity("SayHelloAgain"))
+                {
+                    nestedActivity?.SetTag("bar", "Hello, World!");
+                    nestedActivity?.SetStatus(ActivityStatusCode.Ok);
+                }
+            }
+        }
+
+        private static void GenerateLogs()
+        {
+            using var loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.AddOpenTelemetry(options =>
+                {
+                    options.AddAzureMonitorLogExporter(o => o.ConnectionString = ConnectionString);
+                });
+            });
+
+            var logger = loggerFactory.CreateLogger<Program>();
+            logger.LogInformation("Hello from {name} {price}.", "tomato", 2.99);
+        }
+
+        private static void GenerateMetrics()
+        {
+            Meter MyMeter = new("MyCompany.MyProduct.MyLibrary", "1.0");
```

Comment:
Is Meter disposable?

Reply:
Yes, it is disposable. Like `ActivitySource` it is mostly uses as static member.